docs/dg/dev/acp/apis-and-overview-diagrams-for-payment-service-providers.md

```diff
@@ -45,6 +45,12 @@ The following diagram explains the flow of a headless payment page based on Glue
 
 For information about endpoints and messages, see the following sections.
 
+The following diagram explains the flow of a headless payment page with an express-checkout payment method based on Glue API.
+
+![headless-express-checkout-flow](https://spryker.s3.eu-central-1.amazonaws.com/docs/dg/dev/acp/apis-and-overview-diagrams-for-payment-service-providers.md/headless-express-checkout-flow.png)
```

Comment:
![ExpressCheckoutHeadlessPaymentFlow](https://github.com/user-attachments/assets/e451fef8-7ea4-4407-8f89-f4fab17b5a50)

Reply:
I think it's usable for both parties. Project developers should also know so they can track this part as it it between SCOS and any App.

> they could be different from app to app

They shouldn't differ from App to App. If we have a case where they MUST be different, we can add an additional diagram to the PSP Payment Method that changes them.